packages/adminrouter/docker/detect_ip_public

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+import logging
+import logging.handlers
+import os
+import sys
+import time
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+syslog_handler = logging.handlers.SysLogHandler(address='/dev/log')
+formatter = logging.Formatter('%(module)s.%(funcName)s: [%(levelname)s] %(message)s')
+syslog_handler.setFormatter(formatter)
+logger.addHandler(syslog_handler)
+
+DATAFILE = "/usr/local/detect_ip_public_data.txt"
+
+
+def main():
+    if DATAFILE is None:
+        logger.error("detect IP script data file location unknown, aborting")
+        return 1
+
+    try:
+        with open(DATAFILE, 'r') as fh:
+            control_string = fh.readline().strip()
+    except IOError:
+        logger.error("Failed to open `{}`, aborting!".format(DATAFILE))
+        return 1
+
+    if control_string.startswith("timeout "):
+        try:
+            sleep_time = int(control_string.split()[1])
+        except (IndexError, ValueError):
+            msg_fmt = "Cannot determine sleep time basing on `{}` string"
+            logger.error(msg_fmt.format(control_string))
+            return 1
+
+        logger.info("Sleeping for `{}` seconds".format(sleep_time))
+        time.sleep(sleep_time)
+        return 0
+
+    if control_string.startswith("break with "):
+        try:
+            exit_status = int(control_string.split()[2])
+        except (IndexError, ValueError):
+            msg_fmt = "Cannot determine exit status basing on `{}` string"
+            logger.error(msg_fmt.format(control_string))
+            return 1
+
+        msg = "Exiting with `{}` status".format(exit_status)
+        print(msg)
```

Comment:
I wanted it to simulate script breaking in some realistic way (error message plus exit status), plus it may make debugging easier (e.g. if AR tries to parse output despite non-zero exit status it will break much more verbosely).